contracts/fuelchain/FuelChainConsensus.sol

```diff
@@ -7,21 +8,42 @@ import {AccessControlUpgradeable} from "@openzeppelin/contracts-upgradeable/acce
 import {PausableUpgradeable} from "@openzeppelin/contracts-upgradeable/security/PausableUpgradeable.sol";
 import {CryptographyLib} from "../lib/Cryptography.sol";
 
+/// @notice Structure for commits
+struct Commit {
+    bytes32 blockHash;
+    uint32 timestamp;
+    address reserved1;
+    uint16 reserved2;
+}
+
 /// @notice The Fuel v2 chain state consensus
 contract FuelChainConsensus is Initializable, PausableUpgradeable, AccessControlUpgradeable, UUPSUpgradeable {
     ///////////////
     // Constants //
     ///////////////
 
+    /// @dev The commit proccess parameters
+    //TODO: update these values once block time and commit frequency are finalized
+    uint256 public constant NUM_COMMIT_SLOTS = 240; //30 days worth of commits
+    uint256 public constant BLOCKS_PER_COMMIT_INTERVAL = 10800;
+    uint256 public constant TIME_TO_FINALIZE = 10800;
+
     /// @dev The admin related contract roles
     bytes32 public constant PAUSER_ROLE = keccak256("PAUSER_ROLE");
+    bytes32 public constant COMMITTER_ROLE = keccak256("COMMITTER_ROLE");
```

Comment:
These are only constants so not a big deal, but consider using `CryptographyLib.hash()` for consistency.